### PR TITLE
Feature/ansible filebeat

### PR DESCRIPTION
**Pull request by MinaMohammadi**

### CODE REVIEW

File: roles/filebeat-agent/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   update_cache: true
+   upgrade: dist
+   cache_valid_time: 3600
+   force_apt_get: true
+
+# Installing Filebeat
+
+- name: Install Filebeat
+  apt:
+   name: filebeat
```
Comment (alrzazz) [inline roles/filebeat-agent/tasks/main.yml:31]: specifies version of package

File: roles/filebeat-agent/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- name: Adding Elasticsearch repo
+  apt_repository:
+   repo: "deb {{elasticsearch_repo_base_url}}"
+   state: present
+
+# Updateing Repository
+
+- name: "Update cache & Full system update"
+  apt:
+   update_cache: true
```
Comment (alrzazz) [inline roles/filebeat-agent/tasks/main.yml:22]: update don't need separate task, use update_cache in package installation

Comment (alrzazz) [inline roles/filebeat-agent/tasks/main.yml:22]: update don't need separate task, use update_cache in package installation

File: roles/filebeat-agent/tasks/main.yml (hunk 1 above)
Comment (MinaMohammadi) [reply roles/filebeat-agent/tasks/main.yml]: Added filebeat version 7.17.1

File: roles/filebeat-agent/tasks/main.yml (hunk 2 above)
Comment (MinaMohammadi) [reply roles/filebeat-agent/tasks/main.yml]: done

**Pull request merged**